### PR TITLE
CEM-2611 ThemeSwitch

### PR DESCRIPTION
**Pull request by Zhuk314**
Animation
Black Theme

### CODE REVIEW

**Review by ralph-dev — CHANGES_REQUESTED** [review]
You'll want to transition the Colors that are changing so it doesn't pop the color difference. Ex your going from white to black instead of doing it instantly, do it with a color transition

File: src/Inputs/ThemeSwitch/ThemeSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+interface IDataObject {
+    buttonName: string;
+    icon: StyledIcon;
+    themeSwitchBackgroundColor: string;
+    headerColor: string;
+    buttonsRowColor: string;
+    buttonBackgroundColor: string;
+    buttonTextColor: string;
+    movingDivColor: string;
```
Comment (ralph-dev) [inline src/Inputs/ThemeSwitch/ThemeSwitch.tsx:18]: Try to use a more specific type 
"Cow" would not be correct here

**Review by ralph-dev — CHANGES_REQUESTED** [review]
Typings need to be stricter, Gif needed for PR

**Comment by Zhuk314** [comment]
Animation

**Comment by ralph-dev** [comment]
@Zhuk314 can you provide another story with unequal text length
Ex:

Light,  super super dark, the darkest dark dark you've ever seen

Just as a test for the component :)

**Comment by Zhuk314** [comment]
Animation_ThemeSwitchUnequalText

Animation with Unequal Text.

**Review by ralph-dev — APPROVED** [review]
LGTM

**Pull request merged**